src/pages/migrating/faq.mdx

```diff
@@ -107,27 +107,13 @@ description:
 - The new names imply the usage and so will be much easier to apply. For
   example, `text-03` will be `text-placeholder`.
 
-### How big is the effort to migrate current Sketch designs to the v11 Sketch kit?
+### How big is the effort to migrate from the v10 to v11 Figma kit?
 
-- The Carbon Sketch files will automatically update to v11. Connected styles and
-  symbols will automatically receive the v11 updates once the library update is
-  accepted by the users in each design file.
-
-- If you’re not ready to update to v11, you can stop accepting library updates
-  to that file until you are ready.
-
-- Sketch files for v10 will be available in the GitHub archive but will no
-  longer be an attached library. There is no way to properly version Sketch
-  Cloud files. There is only the latest version.
-
-- The effort to update local/PAL assets depends on the structure of files. If
-  you are using the Carbon layer styles then the update will be automatic. If
-  you’re using hard values and just specing the token names, then you will need
-  to update your specs.
-
-- Read the
-  [Sketch documentation](https://www.sketch.com/docs/libraries/#updating-library-components)
-  for more details.
+The v10 Carbon libraries are separate from the new v11 Carbon libraries. We
+created separate Figma libraries for v10 and v11 based on user feedback for
+migration purposes. The v10 libraries will not have a continuous update to
```

Comment:
We did this more maintainence and version purposes, not migration. We did it so people didn't have to migrate before they were ready to.
```suggestion
versioning purposes. The v10 libraries will not have a continuous update to
```

Reply:
Or maybe its this ... i like this phrasing in another section
```suggestion
migration preferences. The v10 libraries will not have a continuous update to
```